samples/BikeSharingApp/BikeSharingWeb/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bikesharingweb",
+  "version": "1.0.0",
+  "description": "BikeSharing web app",
+  "main": "server.js",
+  "dependencies": {
+    "express": "^4.16.4",
+    "isomorphic-fetch": "^2.2.1",
+    "next": "^7.0.2",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "react-rating": "^1.6.2",
+    "react-responsive": "^6.0.1",
+    "universal-cookie": "^3.0.7"
+  },
+  "devDependencies": {
+    "nodemon": "^1.18.10"
+  },
+  "scripts": {
+    "dev": "nodemon server.js -w server.js -w server -w universal -w next.config.js -w config.js -w package.json",
+    "export": "next export -o html",
+    "start": "next build && NODE_ENV=production node server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ContosoBikeRental/BikeSharingSampleApp.git"
```

Comment:
>https://github.com/ContosoBikeRental/BikeSharingSampleApp.git [](start = 16, length = 61)

These kind of references will be taken care of in a future PR. I would rather avoid mixing this big copy with smaller changes.